mycore-mods/src/main/java/org/mycore/mods/MCRMODSEmbargoCronjob.java

```diff
@@ -83,29 +85,32 @@ public void startUp(ServletContext servletContext) {
     }
 
     private void searchDocumentsToRelease(Consumer<MCRObjectID> objectReleaser) {
-        final SolrClient solrClient = MCRSolrClientFactory.getMainSolrClient();
-        final ModifiableSolrParams params = new ModifiableSolrParams();
-        final LocalDate today = LocalDate.now();
-        final String todayString = today.format(DateTimeFormatter.ISO_DATE);
-
-        params.set("start", 0);
-        params.set("rows", Integer.MAX_VALUE - 1);
-        params.set("fl", "id");
-        params.set("q", "mods.embargo.date:[* TO " + todayString + "]");
-
-        try {
-            final QueryResponse response = solrClient.query(params);
-            Set<MCRFixedUserCallable<Boolean>> releaseCallables = response.getResults().stream()
-                .map(result -> (String) result.get("id"))
-                .map(MCRObjectID::getInstance)
-                .map(id -> new MCRFixedUserCallable<>(() -> {
-                    objectReleaser.accept(id);
-                    return true;
-                }, MCRSystemUserInformation.getSuperUserInstance())).collect(Collectors.toSet());
-
-            EXECUTOR_SERVICE.invokeAll(releaseCallables);
-        } catch (SolrServerException | IOException | InterruptedException e) {
-            LOGGER.error("Error while searching embargo documents!", e);
+        final Predicate<String> stringEmpty = String::isEmpty;
+        if(MCRConfiguration2.getString("MCR.Solr.ServerURL").filter(stringEmpty.negate()).isPresent()){
```

Comment:
empty check is not necessary:
```suggestion
        MCRConfiguration2.getString("MCR.Solr.ServerURL").ifPresent(solrURL ->{
          [code here]
        });
```

Reply:
its for the case somebody has "MCR.Solr.ServerURL=" in his mycore.properties